featuretools/tests/primitive_tests/test_primitive_utils.py

```diff
@@ -227,18 +227,19 @@ def test_check_input_types():
     sem_tag_type_checks = set()
     unique_input_types = set()
     expected_log_in_check = {
-        "boolean_nullable",
```

Comment:
Is there a reason for changing this test?

Reply:
Just listed in types in alphabetical order. I was working on this file when replacing Lag with NumericLag but I can just revert these changes to make this MR less confusing.